lib/src/ui/common/util/notifier_state.dart

```diff
@@ -1 +1,27 @@
 enum NotifierState { initial, loading, loaded, error }
```

Comment:
skoro są osobne klasy do stanu to ten enum powinien raczej wylecieć

Reply:
identyczna sytuacja była wcześniej, ale true, jest zbędny, jedyne co dawał to możliwość użycia switcha

Reply:
ogólnie to nie podoba mi się to rozwiązanie, bo co w sytuacji kiedy mamy jednocześnie załadowane dane i ładowanie/błąd ? wszystkie stany musiałyby mieć wszystkie dane i sprowadza się to do stworzenia jednego wspólnego stanu, który by miał informacje o ładowaniu i błędach

Reply:
tutaj akurat nie mamy takiego przypadku, ale prawda, też mamy u siebie w projektach coś takiego jak wkleiłeś, tu poszedłem za bardzo za ciosem :P można zrobić jak najbardziej w ten sposób